### DISCUSSION TITLE
problem to use api mode

### DISCUSSION THREAD

**Discussion by Remi1945** [Stellarium/stellarium#2359] [Q&A]
Hello
I try to set a new location with the api but it does not work.
I have typed this in my navigator address bar 
`http://localhost:8090/api/location/setlocationfields?latitude=10.0&longitude=20.0&altitude=2000&name=avion&country=france&planet=earth`
The answer is:
**_unsupported operation, GET: list, countrylist, planetlist, planetimage_** 
What is wrong ?

**Comment by alex-w** [comment]
See documentation: https://stellarium.org/doc/0.21/remoteControlApi.html#rcLocationService

The method `setlocationfields` is under section **POST operations** - why you tried use GET method for it?

**Reply by Remi1945** [reply]
Does it  mean you can't use a navigator to do this?

**Reply by alex-w** [reply]
> Does it mean you can't use a navigator to do this?

Standard address bar in any browser can send GET requests only. Please use special tools (e.g. `curl` or `Postman`) to send POST requests.

**Reply by Remi1945** [reply]
Ok I just use the navigator to test. For the final usage I use an other tools which can send POST and GET.
